src/api.js

```diff
@@ -1,16 +1,29 @@
+import { logError } from '@edx/frontend-platform/logging';
 import { examRequiresAccessToken, store } from './data';
 
 export function isExam() {
   const { exam } = store.getState().examState;
+  if (!exam) {
+    return false;
```

Comment:
How is this check different from what's being returned? If there is no exam set in the state, wouldn't it's value be `null`?

Reply:
My understanding was that exam.id wouldn't be able to be accessed because exam itself is null which is what was leading to the error in the case where the unit wasn't an exam.

Here's the original convo for context: https://github.com/openedx/frontend-app-learning/pull/1083#discussion_r1159956063

Reply:
Ahh got it, I might do something like `exam ? exam.id !== null : false` just to make it more concise

Reply:
I'll throw some JS magic in here, the `?.` operator is really useful for null checking a chain of properties. So you can do `exam?.id ? true : false` or even just `!!exam?.id`. `!!` is a common use for 'cast to boolean'.